src/components/ItemsContainer.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+function ItemsContainer() {
+  return (
+    <>
+      <div className="itemsContainer">
+        <button type="button" className="items">
+          AC
+        </button>
+        <button type="button" className="items">
+          +/-
+        </button>
+        <button type="button" className="items">
+          %
+        </button>
+        <button type="button" className="items orangeColor">
+          ÷
+        </button>
+        <button type="button" className="items">
+          7
+        </button>
+        <button type="button" className="items">
+          8
+        </button>
+        <button type="button" className="items">
+          9
+        </button>
+        <button type="button" className="items orangeColor">
+          *
+        </button>
+        <button type="button" className="items">
+          4
+        </button>
+        <button type="button" className="items">
+          5
+        </button>
+        <button type="button" className="items">
+          6
+        </button>
+        <button type="button" className="items orangeColor">
+          -
+        </button>
+        <button type="button" className="items">
+          1
+        </button>
+        <button type="button" className="items">
+          2
+        </button>
+        <button type="button" className="items">
+          3
+        </button>
+        <button type="button" className="items orangeColor">
+          +
+        </button>
+        <button type="button" className="items longLength">
+          0
+        </button>
+        <button type="button" className="items">
+          .
+        </button>
+        <button type="button" className="items orangeColor">
+          =
+        </button>
+      </div>
+    </>
```

Comment:
- [ ] You did a great job adding the calculator UI, but the orange color does not match the given design. Kindly make sure the colors match the design.
- [ ] Kindly remove all the boilerplate files in the src (logo.svg) and in the public (only the index.html file to remain) folders.


![design](https://github.com/Kingno3l/Math-Magician/assets/53124548/679b0664-7abd-4fb9-8d0b-fab64682a614)


![color](https://github.com/Kingno3l/Math-Magician/assets/53124548/978b98a4-ccbc-41ee-90f5-73ba479c6590)